source/_components/mcp23017.markdown

```diff
@@ -0,0 +1,128 @@
+---
+layout: page
+title: "MCP23017 I2C GPIO expander"
+description: "Instructions on how to integrate the MCP23017 GPIO pin expander with I2C interface into Home Assistant."
+date: 2019-04-14 07:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: raspberry-pi.png
+ha_category:
+  - DIY
+  - Binary Sensor
+  - Switch
+ha_release: 0.92
+ha_iot_class: Local Polling
+redirect_from:
```

Comment:
Please remove, this is only for existing pages.

Reply:
Yet you have solved it well 😉 

`redirect_from` is only needed if old / existing pages have to be referred to a new page, for example after merging single platform pages to 1 page and therefore old pages are removed and no longer exist.